rdr_service/dao/genomics_dao.py

```diff
@@ -501,6 +501,24 @@ def get_collection_tube_max_set_id(self):
                 GenomicSetMember.genomicWorkflowState == GenomicWorkflowState.CONTROL_SAMPLE
             ).one()
 
+    def get_member_count_from_manifest_path(self, filepath):
+        """
+        Retrieves the count of members based on file path
+        :return: integer
+        """
+        with self.session() as s:
+            return s.query(
+                functions.count(GenomicSetMember.id)
+            ).join(
+                GenomicFileProcessed,
+                GenomicFileProcessed.id == GenomicSetMember.aw1FileProcessedId
+            ).join(
+                GenomicManifestFile,
+                GenomicManifestFile.id == GenomicFileProcessed.genomicManifestFileId
+            ).filter(
+                GenomicManifestFile.filePath == filepath
```

Comment:
We'll want to filter on `GenomicSetMember.genomicWorkflowState != GenomicWorkflowState.IGNORE`